scripts/build.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+
+const fs = require('fs');
+const del = require('del');
+const rollup = require('rollup');
+const babel = require('rollup-plugin-babel');
+const eslint = require('rollup-plugin-eslint');
+const uglify = require('rollup-plugin-uglify');
+const pkg = require('../package.json');
+
+const bundles = [
+  {
+    format: 'umd',
+    dest: 'build/stardog.min.js',
+    plugins: [
+      eslint(),
+      babel({
+        exclude: 'node_modules/**',
+        presets: ['es2015-rollup'],
+      }),
+      uglify(),
+    ],
+    moduleName: 'stardog',
+    minify: true,
+  },
+  {
+    format: 'umd',
+    dest: 'build/stardog.umd.js',
+    plugins: [
+      eslint(),
+      babel({
+        exclude: 'node_modules/**',
+        presets: ['es2015-rollup'],
+      }),
+    ],
+  },
+  {
+    format: 'es',
+    dest: 'build/stardog.esm.js',
+    plugins: [
+      eslint(),
+      babel({
+        exclude: 'node_modules/**',
+        presets: ['es2015-rollup'],
+      }),
+    ],
+  },
+];
+
+Promise.resolve()
+  .then(() => del(['build/*']))
+  .then(() => {
+    bundles.forEach((bundle) => {
+      rollup.rollup({
+        entry: 'js/stardog',
+        external: Object.keys(pkg.dependencies),
+        plugins: bundle.plugins,
+      })
+        .then(build => build.write({
+          dest: bundle.dest,
+          format: bundle.format,
+          moduleName: bundle.moduleName || 'main',
+          sourceMap: !bundle.minify || bundle.sourceMap || false,
+        }))
+        .then(() => console.log(`${bundle.dest} written successfully.`));
+    });
+  })
+  .then(() => {
+    delete pkg.devDependencies;
+    delete pkg.scripts;
+    pkg.main = 'stardog.min.js';
```

Comment:
Since these filenames don't change, you could just update the real `package.json`. Also, look into the `browser` field as well.